.nucleus.yaml

```diff
@@ -34,10 +34,7 @@ branches:
     winter25:
         pull-request:
             <<: *branch-definition
-    spring24:
-        pull-request:
-            <<: *branch-definition
-    summer24:
+    spring25:
```

Comment:
Dunno why we had 3 here. We only need two since 252 (Winter '25) is fully released now.